packages/bitcore-wallet-service/src/lib/chain/eth/index.ts

```diff
@@ -264,7 +283,7 @@ export class EthChain implements IChain {
       for (let index = 0; index < recipients.length; index++) {
         let params = {
           ...recipients[index],
-          nonce: Number(txp.nonce) + Number(index),
+          nonce: txp.nonce ? Number(txp.nonce) + Number(index) : null,
```

Comment:
This and above should be `txp.nonce != null ? ...` so that if txp.nonce == 0 it doesn't set it to null.